packages/website/tsconfig.json

```diff
@@ -12,7 +12,14 @@
             "@styles/*": ["./src/styles/*"]
         },
         "lib": ["ES2020", "dom"],
-        "typeRoots": ["./types", "./node_modules/@types"]
+        "typeRoots": ["./types", "./node_modules/@types"],
+        "strict": false,
+        "forceConsistentCasingInFileNames": true,
+        "noEmit": true,
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "jsx": "preserve"
     },
-    "include": ["src", "docs"]
+    "include": ["src", "docs"],
+    "exclude": ["node_modules"]
```

Comment:
This is probably not necessary, node_modules doesn't exist inside `src` or `docs`

```suggestion
```